contracts/ownership/Heritable.sol

```diff
@@ -50,6 +50,22 @@ contract Heritable is Ownable {
     heir = newHeir;
   }
 
+  /**
+   * @dev Use these getter functions to access the internal variables in
+   * an inherited contract.
+   */
+  function getHeir() public view returns(address) {
```

Comment:
As of #666 we've adopted the convention that private state variables with public getters follow the naming convention that

- state variable has a suffix underscore `_`, and
- getter is the name of the variable without suffix underscore.

In this case, for example, we'd have `address private heir_` and `function heir() public view`.

Can you please change the PR to follow this convention?

Reply:
Will do.